frontend/src/scenes/insights/ActionFilter/ActionFilter.tsx

```diff
@@ -208,46 +190,39 @@ export const ActionFilter = React.forwardRef<HTMLDivElement, ActionFilterProps>(
                                     {...commonProps}
                                 />
                             ))}
-                        </SortableContainer>
+                        </SortableActionFilterContainer>
                     ) : (
                         localFilters.map((filter, index) => (
                             <ActionFilterRow
                                 filter={filter as ActionFilterType}
                                 index={index}
                                 key={index}
                                 singleFilter={singleFilter}
-                                showOr={showOr}
                                 hideFilter={hideFilter || readOnly}
-                                horizontalUI={horizontalUI}
                                 filterCount={localFilters.length}
                                 showNestedArrow={showNestedArrow}
                                 {...commonProps}
                             />
                         ))
                     )
                 ) : null}
-                {(!singleFilter || customActions) && (
-                    <div
-                        className={clsx(buttonType !== 'link' ? 'mt' : 'mt-05')}
-                        style={{ display: 'flex', alignItems: 'center' }}
-                    >
+                {!singleFilter && (
+                    <div className="ActionFilter-footer">
                         {!singleFilter && (
-                            <Button
+                            <LemonButton
                                 type={buttonType}
```

Comment:
This button is tall compared to series/step rows and "Add filter group", can we align it by making it `compact`?
<img width="1005" alt="Screen Shot 2022-05-12 at 13 12 12" src="https://user-images.githubusercontent.com/4550621/168060560-631b3a7c-ee90-4eac-9234-78e680ea1cbc.png">
```suggestion
                                type={buttonType}
                                compact
```

Reply:
Good call 👍

Reply:
Assuming you meant `size="small"`

Reply:
Ohh, right, that was renamed 😅

Reply:
Also maybe we could make the "+ Add graph series" button `fullWidth`? 🤔 At least in the above screenshot case that would work better, but not sure about other contexts

Reply:
I tried it out but I think it looks okay. I think it is worth doing and we can revert later. Especially as in the default state this only really effects the hovered/focused state so its barely noticeable